notebooks/official/pipelines/google_cloud_pipeline_components_TPU_model_train_upload_deploy.ipynb

```diff
@@ -710,21 +672,22 @@
         "id": "taskpy_contents"
       },
       "source": [
-        "#### Training script\n",
+        "### Create the training script\n",
+        "\n",
+        "In the next cell, write the contents of the training script to `train.py`. \n",
         "\n",
-        "In the next cell, you write the contents of the training script, `train.py`. In summary:\n",
+        "In summary, you training script does the following:\n",
         "\n",
-        "- Get the directory where to save the model artifacts from the environment variable `AIP_MODEL_DIR`. This variable is set by the training service.\n",
+        "- Gets the directory where to save the model artifacts from the environment variable `AIP_MODEL_DIR`. This variable is set by the training service.\n",
         "- Loads CIFAR10 dataset from TF Datasets (tfds).\n",
         "- Builds a model using TF.Keras model API.\n",
         "- Compiles the model (`compile()`).\n",
         "- Sets a training distribution strategy according to the argument `args.distribute`.\n",
         "- Trains the model (`fit()`) with epochs and steps according to the arguments `args.epochs` and `args.steps`\n",
         "- Saves the trained model (`save(MODEL_DIR)`) to the specified model directory.\n",
-        "\n",
-        "TPU specific changes are listed below:\n",
-        "- Added a section that finds the TPU cluster, connects to it, and sets the training strategy to TPUStrategy\n",
-        "- Added a section that saves the trained TPU model to the local device, so that it can be saved to the AIP_MODEL_DIR"
+        "- Runs TPU specific tasks:\n",
+        "    - Finds the TPU cluster, connects to it, and sets the training strategy to TPUStrategy.\n",
+        "    - Saves the trained TPU model to the local device, so that it can be saved to the `AIP_MODEL_DIR`."
```

Comment:
- change: "Saves the trained TPU model to the local device, so that it can be saved to the `AIP_MODEL_DIR`"
- to: "Saves the trained TPU model to the local device so that the model can be saved to the `AIP_MODEL_DIR`"